src/PermissionRegistrar.php

```diff
@@ -104,6 +104,16 @@ public function forgetCachedPermissions()
         return $this->cache->forget(self::$cacheKey);
     }
 
+    /**
+     * Clear class permissions.
+     */
+    public function clearClassPermissions()
+    {
+        $this->permissions = null;
+
+        return true;
```

Comment:
Is a `return` value needed here? Let's exclude it if it's not really required.

(I know `registerPermissions()` returns `bool`, so maybe you were just following that pattern, but that's an artifact that was kept as part of the public method for backward compatibility amidst other changes. 
I plan to remove it in the next major release.)

Reply:
The PermissionRegistrar class is only initialized once when swoole loads, so setting the protected var doesn't help. I thought the same but after I sprinkled \Log::debug all over the place I saw this behavior. 

I am thinking that this behavior might be related to the use of the singleton as I don't see it elsewhere in __construct in my app models. It seems like the singleton is reused even if the PermissionServiceProvider boots on every request.

I did add the return to match the pattern, it is not needed, I will remove.

Reply:
Thanks!